dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/resources/internal/JAXRS21FrontendMessages.nlsprops

```diff
@@ -0,0 +1,12 @@
+###############################################################################
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+
+continuation.not.supported=Continuation not supported. Please ensure that all servlets and servlet filters support async operations
```

Comment:
Ensure that all servlets and servlet filters support async operations.

Reply:
done